oap-server/server-library/library-integration-test/src/main/java/org/apache/skywalking/oap/server/library/it/ITVersions.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.library.it;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * ITVersions is a utility class to read the version information from the environment file.
+ */
+public class ITVersions {
+
+    static Map<String, String> ENV = readingEnv("test", "e2e-v2", "script", "env");
+
+    // The prefix of the integration version in the environment file.
+    static final String INTERGRATION_PREFIX = "INTEGRATION_";
```

Comment:
I think we should load all `SW_` prefix, and then we could reuse `SW_BANYANDB_COMMIT` directly, rather than redeclaring a new one.

Reply:
Do you mean that we won't have a customized version for IT and can use the same version directly?